src/communication/communication.hpp

```diff
@@ -20,8 +20,7 @@ class Communication {
 
   void sendOrientation(Vec3f orientation);
   void sendIMURawData(Vec3i accelerometer, Vec3i gyroscope, Vec3i magnetometer);
-  void sendIMUData(Vec3f acceleration, Vec3f angular_velocity,
-                   Vec3f magnetic_field);
+  void sendIMUData(Vec3f acceleration, Vec3f angular_velocity, Vec3f magnetic_field);
```

Comment:
Use clang formatter, Function signature is too big to be written in a single line